docker-compose.yml

```diff
@@ -1252,6 +1220,12 @@ services:
       - ACCEPT_EULA=Y
       - SA_PASSWORD=Strong!Passw0rd
 
+### PRE PULL TEST CONTAINER IMAGES BELOW HERE TO GET THEM INTO THE VM TO MAKE IT GO FASTER ###
```

Comment:
When we build our VMs we will pull these containers and store them on the VM, we don't use them though from this docker compose.